Example/Example Swift/ComponentAPIDocumentsService.swift

```diff
@@ -15,9 +15,11 @@ final class ComponentAPIDocumentsService: ComponentAPIDocumentServiceProtocol {
     var partialDocuments: [String: PartialDocumentInfo] = [:]
     var compositeDocument: GINIDocument?
     var analysisCancellationToken: BFCancellationTokenSource?
+    var documentMetadata: GINIDocumentMetadata?
     
-    init(sdk: GiniSDK) {
+    init(sdk: GiniSDK, documentMetadata: GINIDocumentMetadata?) {
```

Comment:
Since document metadata is optional, I would add it as a property or create a second constructor. This would keep backwards compatibility, too.